### DISCUSSION TITLE
Welcome to Nillion's Github Discussions Forum

### DISCUSSION THREAD

**Discussion by oceans404** [NillionNetwork/discussions#1] [Announcements]
# 🎉 Welcome to Nillion's Discussions Forum! 🎉

This forum is designed to be a central hub for technical discussions related to Nillion, where builders can share ideas, ask questions, showcase their projects, and stay updated on the latest news and announcements from the Nillion team.

To help keep the discussions organized and easy to navigate, we've set up several categories with specific purposes:

### 📣 [Announcements](https://github.com/orgs/NillionNetwork/discussions/categories/announcements?discussions_q=)
This category is where you'll find all the latest updates and announcements from the Nillion team. Whether it's new features, product releases, or important news, you'll find it all here.

### 💬 [General](https://github.com/orgs/NillionNetwork/discussions/categories/general?discussions_q=)
Want a place to chat about anything and everything related to Nillion? This is it! Use general to share your thoughts, discuss MPC topics, or just hang out with fellow community members.

### 💡 [Ideas](https://github.com/orgs/NillionNetworkdiscussions/categories/ideas?discussions_q=)
Got a brilliant idea for a new network or SDK feature? Want to provide feedback on existing features? Use the ideas category to share your ideas, suggestions, and feedback to help shape the future of Nillion.

### 🙏 [Q&A](https://github.com/orgs/NillionNetwork/discussions/categories/q-a?discussions_q=)
Stuck on a Nillion concept? Need help debugging your code or using the SDK? Don't worry, we've got you covered! Ask questions, seek advice, and get help from the community in this Q&A-focused category.

### 💻 [Show and Tell](https://github.com/orgs/NillionNetwork/discussions/categories/show-and-tell?discussions_q=)
Have you built something awesome with Nillion that you're proud of? Whether it's a cool project, a useful tool, or a creative application, we want to see it! Share your creations, showcase your work, and inspire others in this show and tell category.

## 🤝 Community

We're committed to fostering a welcoming and inclusive community here on the Nillion discussions forum, so please be respectful and considerate of your fellow community members. Let's work together to make this a vibrant and valuable resource for everyone involved.

### 🤿 Next steps

Immerse yourself in the discussions and actively participate by offering assistance to fellow community members. Let's collaborate to create remarkable, high-value, data-focused applications that push boundaries and inspire innovation!

**Comment by szupion** [comment]
Good afternoon, do you have any plans to launch a community activity like an ambassador programme or work tasks?

**Reply by oceans404** [reply]
hi there! this sounds awesome! I'd love to hear more about the type of ambassador program you'd like to see from Nillion. Can you add it in Ideas: https://github.com/orgs/nillion-oss/discussions/categories/ideas

**Comment by kkk2791** [comment]
How can I get involved in the development of the project?

**Reply by oceans404** [reply]
we're hiring! check out open roles across engineering, ops, and marketing: https://apply.workable.com/nillion/

**Comment by tangocodes** [comment]
Is there any open-source contribution for the project

**Comment by dkeseloff** [comment]
Is comprehensive offensive security testing needed?

I have an associate who is an expert white hat hacker, with govt clearance, hired to pwn agencies (with their written permission, of course!).  He’s intrigued by the tech and would like to test its hackability, but only with written consent.

The focus would be on whether a complex, orchestrated attack could compromise the ecosystem on any level, from the kernel up.

I can discuss further, then make intros, if needed.

**Comment by rvzoros** [comment]
Hi

**Comment by szupion** [comment]
hello

чт, 13 мар. 2025 г. в 13:12, rvzoros ***@***.***>:

**Comment by codemintous** [comment]
hii
I encountered a CORS policy issue while making a request to the Nillion API from the content.js file in a Chrome extension. The request is blocked, and the console logs the following error:
Access to fetch at 'https://nilai-a779.nillion.network/v1/chat/completions' from origin 'https://mail.google.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Comment by Network652** [comment]
Hello 👋 
I just need $2000
To start I investment 🙏

**Comment by Helbi10** [comment]
Bagaimana mekanisme 'Nil Message Compute (NMC)' bekerja dalam menjaga kerahasiaan data, terutama dalam skenario penggunaan yang kompleks?

**Comment by johnable72** [comment]
Integration of valuables is needed to this project and how best do I know what I can earn
